### PR TITLE
Adding unit tests for backup utility functions

### PR DESCRIPTION
**Pull request by sahare**
Signed-off-by: sahare <sahare@redhat.com>

Latest coverage:

Screen Shot 2021-08-16 at 9 48 46 AM

### CODE REVIEW

File: controllers/suite_test.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
 	backupv1beta1 "github.com/open-cluster-management-io/cluster-backup-operator/api/v1beta1"
+	restorev1beta1 "github.com/open-cluster-management-io/cluster-backup-operator/api/v1beta1"
```
Comment (sdminonne) [inline controllers/suite_test.go:34]: Reason for this?

Comment (sahare) [reply controllers/suite_test.go:34]: trying to test if adding restorev1beta1 into the scheme works with no error

Comment (sdminonne) [reply controllers/suite_test.go:34]: yeah but the test should be functional: my understanding is that testing the idempotency of scheme adding (even with different import alias) should not stay in our operator.

**Review by sdminonne — APPROVED** [review]
LGTM

**Pull request merged**